lms/djangoapps/discussion_api/api.py

```diff
@@ -48,18 +48,6 @@
 from openedx.core.djangoapps.course_groups.cohorts import get_cohort_id
 
 
-def _get_course_or_404(course_key, user):
```

Comment:
Please revert the changes in this file.  
Only the implementation of `_get_course_or_404` should be updated to call `DiscussionTab.is_enabled`.